packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -250,6 +275,30 @@ const ActivityDetailsSidebar = (props: Props) => {
                 value={selectedUsers}
                 multiple={false}
               />
+
+              {showOrgPicker && (
+                <>
+                  <div className='text-gray-700 my-4 text-sm font-semibold'>Organization</div>
+                  <Select onValueChange={(orgId) => setSelectedOrgId(orgId)} value={selectedOrgId}>
+                    <SelectTrigger className='bg-white'>
+                      <SelectValue />
+                    </SelectTrigger>
+                    <SelectContent>
+                      <SelectGroup>
+                        {viewerOrganizations
+                          .filter((org) =>
+                            mutualOrgsIds.length ? mutualOrgsIds.includes(org.id) : true
+                          )
```

Comment:
+1 This has the same result as [this logic](https://github.com/ParabolInc/parabol/pull/8712/files#diff-b9436085c8610b2c6e1919f190475b18559cbbd0a80bc8f229eb740ed177ada2R148) - let's just use `mutualOrgs` here instead of filtering on `viewerOrganizations` a second time.